cmd/commands/function.go

```diff
@@ -137,6 +137,7 @@ From then on you can use the sub-commands for the 'service' command to interact
 				}
 			} else {
 				printSuccessfulCompletion(cmd)
+				fmt.Fprintf(cmd.OutOrStdout(), "Issue `riff service status %s` to see the status of the function\n", fnName)
```

Comment:
Should this be printed *only* if not using `--wait/--verbose`?

Reply:
@ericbottard Yes. Good catch. Fixed via force push.